dask_geopandas/morton_distance.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import pandas as pd
+from dask_geopandas.hilbert_distance import _continuous_to_discrete_coords
+
+# Based on #http://graphics.stanford.edu/~seander/bithacks.html#InterleaveBMN
```

Comment:
I would move this comment to the function that does the bit-shifting (since it's only that part that is based on that)

Reply:
Updated in 27ca9e7